apps/happy-blocks/package.json

```diff
@@ -0,0 +1,42 @@
+{
+	"name": "upgrade-block",
+	"version": "0.1.0",
+	"description": "A Gutenberg block to show your pride! This block enables you to type text and style it with the color font Gilbert from Type with Pride.",
```

Comment:
I think we should be able to remove any arbitrary stuff from the `package.json` that we can fill out later.